src/language/i18n.js

```diff
@@ -5,6 +5,12 @@ import { i18n } from '@lingui/core'
 import { Helmet } from 'react-helmet'
 import { messages as defaultMessages } from './locales/en/catalog'
 
+export const localeFiles = {
+  de: () => import(`./locales/de/catalog.js`),
+  en: () => import(`./locales/en/catalog.js`),
+  'es-AR': () => import(`./locales/es-AR/catalog.js`),
+}
+
 // This array should equal the array set in .linguirc
 export const locales = ['de', 'en', 'es-AR', 'es', 'it', 'he', 'ro', 'ru', 'vi', 'zh-CN', 'zh-TW', 'ko', 'ja']
 export const defaultLocale = 'en'
```

Comment:
why is "getInitialLocale" doing nothing? it should detect user preferred locale from device web/native